clang/include/clang/Tooling/Inclusions/IncludeStyle.h

```diff
@@ -151,6 +151,21 @@ struct IncludeStyle {
   /// before any other include.
   /// \version 10
   std::string IncludeIsMainSourceRegex;
+
+  /// Character to consider in the include directives for the main header.
```

Comment:
It should also be added to `FormatStyle::operator==` in `Format.h`.

Reply:
Done.